docs/source/options/general_encryption_options.rst

```diff
@@ -13,6 +13,12 @@ General encryption options
 --clear_lead <seconds>
 
     Clear lead in seconds if encryption is enabled.
+    Shaka Packager does not support partial encrypted segments, so if 
+    segment_duration is greater than clear_lead, then only the 
+    first segment is in clear (without encryption), with all 
+    the following segments encrypted. In the opposite way, if your 
+    segment is lower than clear_lead it will not be encrypted.
```

Comment:
The last sentence is a bit difficult to understand. How about changing the whole paragraph to something like this:

Shaka Packager does not support partial encrypted segments, all the segments
including the partial segment overlapping with the initial `clear_lead` seconds are
not encrypted, with all the following segments encrypted. For example, if
`segment_duration` is greater than `clear_lead`, then the first segment is not
encrypted.
then only the first segment is